gateway/uplink.go

```diff
@@ -119,6 +118,7 @@ func matchDeviceAddr(devAddr []byte, devices map[string]*node.Node) (*node.Node,
 }
 
 func decodePayload(ctx context.Context, fPort uint8, path string, data []byte) (map[string]interface{}, error) {
+	//nolint:gosec
```

Comment:
same here, can add to `disable` section, but it's up to your discretion if you think this case is an exception, or if you'd always want to disable this. when I was modularizing the builtins, if I ran into lint issues like this I would check if it was disabled in the rdk `golangci.yaml` file, and if so I would disable it in the module as well

Reply:
I will disable this, thanks!